src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements Drawable, AreaCalculable {
```

Comment:
Don't use verbs for class/interface names. The interface name 'Drawable' should be noun-based, for example, 'DrawableElement'.